### PR TITLE
Add readme, typography and interchanged index and doc.html

### PR DESCRIPTION
**Pull request by VLeads**
Add Readme
Add Typography

interchanged doc.html and index.html file as main home page code was in doc.html previously. 

Fixed some CSS issues

Gave same height to all code snippets.

### CODE REVIEW

File: doc/doc.css [hunk 1]
```diff
@@ -1,44 +1,226 @@
...
+    margin-right: 6 rem;
+}
+
+.header__links li i{
+    color: black;
+}
 
+.header__links li i:hover{
+    cursor: pointer;
+    /* color: var(--primary-dark-color) */
```
Comment (romabulani) [inline doc/doc.css]: Remove the comments if not required

File: doc/doc.css [hunk 2]
```diff
@@ -1,44 +1,226 @@
...
+    border: 1px solid rgb(20 20 20 / 12%);
+    /* overflow-y: auto; */
+    padding: 1rem;
+    padding-top: 3rem;
+    padding-bottom: 3rem;
+    display: flex;
+    /* flex-direction: column; */
+    /* justify-content: space-evenly; */
+    /* align-items: center; */
+    flex-wrap: wrap;
```
Comment (romabulani) [inline doc/doc.css]: Here also remove comments if not required

File: styles.css [hunk 3]
```diff
@@ -1,215 +1,39 @@
...
+    height: 4.5rem;
     font-size: 1.8rem;
-    color: var(--dark-color);
-    margin-top: 0.5rem;
-    margin-bottom: 0.5rem;
-    padding: 0.5rem;
-    display: grid;
-    justify-content: center;
+    text-align: center;
+    line-height: 24px;
```
Comment (romabulani) [inline styles.css:28]: If possible, Use rems here to maintain consistency

File: doc/doc.html [hunk 4]
```diff
@@ -45,24 +49,889 @@
...
+            <h3>Rounded Image</h3>
+            <p>You can add class img-round to make your image round shaped.</p>
+
+            <div class="component-example__container align-center">
+              <img
+                class="img-round"
+                src="../assets/images/img-responsive.jpg"
+                alt=""
+                loading="lazy"
+              />
```
Comment (romabulani) [inline doc/doc.html]: Write alt value here

File: index.html [hunk 5]
```diff
@@ -45,961 +41,22 @@
...
-              "
-              sandbox="allow-scripts allow-same-origin"
-            >
-            </iframe>
-          </div>
-        </section>
-      </main>
+        <div class="hero-right">
+          <img src="./assets/images/hero.png" alt="" />
+        </div>
```
Comment (romabulani) [inline index.html]: Add value to alt attribute

**Review by romabulani — APPROVED** [review]
Good to merge after the suggested changes

File: doc/doc.css [hunk 6]
```diff
@@ -1,44 +1,226 @@
...
+    
+}
+
+.sidebar{
+    padding: 3rem;
+    padding-left: 4.5rem;
+    padding-right: 3rem;
+    position: sticky;
+    overflow-y: auto;
+    top: 7.5rem;
```
Comment (znjs) [inline doc/doc.css:76]: top property doesn't work for static positioning, you can remove it

**Review by mnath30 — APPROVED** [review]
Great UI! Everything looks good.
Just a small suggestion: You can arrange the CSS properties in alphabetical order. Also, if any properties are not required you can remove those instead of commenting those out in the code.

File: doc/doc.css [hunk 7]
```diff
@@ -1,44 +1,226 @@
...
+.sidebar li{
+    list-style: none;
+}
+
+.sidebar a:hover{
+    color: var(--primary-light-color);
+}
+
+/* ****** scroll margin ***** */
+ #alert-section,
```
Comment (znjs) [inline doc/doc.css:131]: if possible try targeting using `sidebar__link >a` this can reduce the number of lines

**Review by znjs — APPROVED** [review]
Clean UI and liked the color contrast

File: doc/doc.css (hunk 1 above)
Comment (VLeads) [reply doc/doc.css]: Done

File: doc/doc.css (hunk 2 above)
Comment (VLeads) [reply doc/doc.css]: Done

File: styles.css (hunk 3 above)
Comment (VLeads) [reply styles.css:28]: Yups. Changed 👍

File: doc/doc.html (hunk 4 above)
Comment (VLeads) [reply doc/doc.html]: Done

File: index.html (hunk 5 above)
Comment (VLeads) [reply index.html]: done

File: doc/doc.css (hunk 6 above)
Comment (VLeads) [reply doc/doc.css:76]: It is creating problem after removing "top". That's why I have used it here.

**Comment by VLeads** [comment]
> Great UI! Everything looks good. Just a small suggestion: You can arrange the CSS properties in alphabetical order. Also, if any properties are not required you can remove those instead of commenting those out in the code.

Thankyou :) Alright, I'll do that.

**Pull request merged**